rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -172,6 +182,11 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    Date-Time:
+      description: the date-time notation as defined by RFC 3339, section 5.6
+      type: string
+      format: date-time
```

Comment:
✏️ I think you can remove the "RFC 3339, section 5.6 date-time format" parts in the descriptions because that is already documented with this line.